docs/source/quick_start.rst

```diff
@@ -0,0 +1,191 @@
+
+        **Install Prerequisites**
+
+        Install the Rust build tools, other essential build tools, and GTK 4 headers:
+
+        .. tab:: Debian/Ubuntu
+
+            .. code::
+
+                apt install rustc cargo build-essential libgtk-4-dev git
+
+        .. tab:: Fedora
+
+            .. code::
+
+                yum install rust cargo make gcc gcc-c++ gtk4-devel pango-devel git
+
+        Note that Packetry requires a minimum Rust version of 1.74. If your distribution's packages are older than this, use `rustup <https://rustup.rs/>`__ to get the latest Rust toolchain and manage your Rust installation.
+
+        **Download Packetry Source Code**
+
+        Clone the Packetry repository:
+
+        .. code::
+
+            git clone https://github.com/greatscottgadgets/packetry.git
+
+        **Build and Install Packetry**
+
+        .. code::
+
+            cd packetry
+            cargo build --release
+            sudo cp target/release/packetry /usr/local/bin
+
+        Hint: You can copy the ``packetry`` executable to any directory in your ``PATH`` instead of ``/usr/local/bin``. If you've used ``rustup`` and have ``~/.cargo/bin`` in your ``PATH``, ``~/.cargo/bin`` is a great choice!
+
+        Note: Do not build with ``--all-features``. All the optional features currently in the package are for debug/test purposes only, and will prevent or degrade normal use of the application if enabled. See ``Cargo.toml`` for details.
+
+.. tab:: macOS
+
+    .. tab:: Binary
+
+        Homebrew installation coming soon! For now you can build and install with Cargo.
+
+    .. tab:: Cargo
+
+        **Install Prerequisites**
+
+        Use `rustup <https://rustup.rs/>`__ to install the latest Rust toolchain and manage your Rust installation.
+
+        Use `Homebrew <https://brew.sh/>`__ to install GTK 4:
+
+        .. code::
+
+            brew install gtk4
+
+        **Download Packetry Source Code**
+
+        Clone the Packetry repository:
+
+        .. code::
+
+            git clone https://github.com/greatscottgadgets/packetry.git
+
+        **Build and Install Packetry**
+
+        .. code::
+
+            cd packetry
+            cargo build --release
+            cp target/release/packetry ~/.cargo/bin
+
+        Hint: You can copy the ``packetry`` executable to any directory in your ``PATH`` instead of ``~/.cargo/bin``. If you've used ``rustup``, ``~/.cargo/bin`` should be in your ``PATH``. Otherwise ``/usr/local/bin`` is a great choice!
+
+        Note: Do not build with ``--all-features``. All the optional features currently in the package are for debug/test purposes only, and will prevent or degrade normal use of the application if enabled. See ``Cargo.toml`` for details.
+
+.. tab:: Windows
+
+    .. tab:: Binary
+
+        Download and run the Windows installer from the `latest release <https://github.com/greatscottgadgets/packetry/releases/latest>`__.
+
+    .. tab:: Cargo
+
+        **Install Prerequisites**
+
```

Comment:
I suggest we add a note to indicate that there are other ways one can get the prerequisites as we don't specifically require vcpkg:
```suggestion

Building Packetry requires that the GTK 4 libraries and header files are available, and discoverable via the `pkg-config` mechanism. There are several ways you can arrange this; if in doubt we suggest using [vcpkg](https://vcpkg.io/en/) as follows:

```

Reply:
Is there a reason to treat this differently than the other "Install Prerequisites" sections? We're documenting our recommended and supported process, not every possible way to do things.

Reply:
I was thinking that we may later add a Development section to the documentation and potentially move all the "From source code" installation instructions and most of the current README contents there. Then we could make those instructions more verbose, and the quick start installation instructions could direct folks to the Development section if they prefer to install from source.

Reply:
I think there is a good reason to mention this specifically on Windows, which is that there are a lot of projects that are very tied into the vcpkg ecosystem (e.g. have vcpkg as a submodule, and require it to build successfully). I'd like it to be clear that we're not one of those projects; as long as people have GTK discoverable by `pkg-config` somehow, things should work. That means that e.g. people who already have MSYS2 set up can just use that instead.

Reply:
To be honest, I'm not really sure if vcpkg is the best thing to recommend in the first place. It requires building everything from source, which takes a lot of time and disk space. I'm not keen on recommending MSYS2 though, because it's a lot more more invasive whereas vcpkg is self-contained.

Another option would be to suggest the binary packages from https://github.com/wingtk/gvsbuild, but I haven't tested that option myself yet.